cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -95,8 +66,9 @@ def test_power_off_cancel(self, test_vm, verify_vm_running, soft_assert):
         test_vm.wait_for_vm_state_change(desired_state=test_vm.STATE_ON, timeout=720)
         test_vm.power_control_from_cfme(option=test_vm.POWER_OFF, cancel=True)
         if_scvmm_refresh_provider(test_vm.provider)
+        # TODO: assert no event.
```

Comment:
This test does not check for events, so it will be probably good to implement *no event* assertion. I want to do that separately.